frontend-react/src/AppRouter.tsx

```diff
@@ -128,32 +125,36 @@ export const appRoutes: RouteObject[] = [
                                 children: [
                                     {
                                         path: "",
-                                        element: <DocumentationPage />,
+                                        lazy: lazyRouteMarkdown(
+                                            "content/resources/reportstream-api/documentation/Documentation"
+                                        ),
                                         index: true,
                                         handle: {
                                             isContentPage: true,
                                         },
                                     },
                                     {
                                         path: "data-model",
-                                        element: <DataModelPage />,
+                                        lazy: lazyRouteMarkdown(
+                                            "content/resources/reportstream-api/documentation/data-model/DataModel"
+                                        ),
                                         handle: {
                                             isContentPage: true,
                                         },
                                     },
                                     {
                                         path: "responses-from-reportstream",
-                                        element: (
-                                            <ResponsesFromReportStreamPage />
+                                        lazy: lazyRouteMarkdown(
+                                            "content/resources/reportstream-api/documentation/ResponsesFromReportStream"
                                         ),
                                         handle: {
                                             isContentPage: true,
                                         },
                                     },
                                     {
                                         path: "sample-payloads-and-output",
-                                        element: (
-                                            <SamplePayloadsAndOutputPage />
+                                        lazy: lazyRouteMarkdown(
```

Comment:
Just curious, do we know how much time / size we save with the code splitting?

Reply:
And second thought: should we be lazily loading all of our routes?

Reply:
Time to build? No change there. For size, currently only 1-2 kb but could change dramatically as more routes are lazy loaded (and thus library dependencies shift from the main bundle).

Reply:
All routes should likely be shifted to lazy loading, yes.

Reply:
Time to build? No change there. For size, currently only 1-3 kb (gzipped) per but could change dramatically as more routes are lazy loaded (and thus library dependencies shift from the main bundle). All routes should likely be shifted to lazy loading, yes.